### PR TITLE
Improve UI

### LINKED ISSUE #56: Improve UI

### PR DESCRIPTION
**Pull request by Devanshshah1309**
## Changes

1. Mono-space font for entering command
2. New LightTheme instead of dark theme
3. Moved contacts panel on the right

### Intended Behaviour
1. Long team names are not wrapped - just truncated with ellipses.
2. Everything inside the selected team details is shown in full - never truncated (since the whole point is to show all details) 

### LightTheme UI
Screenshot 2022-10-21 at 10 10 18 AM

### Testing
1. Please test that the panels still look good when you have long team names, very long task lists, very long addresses, etc.
4. Please shrink/grow your application size and make sure nothing breaks.

Fixes #56

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,126 +6,140 @@ title: User Guide
 TruthTable is a **desktop app for managing software engineering teams, optimized for use via a Command Line Interface**
 (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, TruthTable can get your task management done faster than traditional GUI apps.
 
-* Table of Contents
+- Table of Contents
   {:toc}
 
---------------------------------------------------------------------------------------------------------------------
+---
```
Comment (marcuspang) [inline docs/UserGuide.md:12]: Can you revert the changes for this file

File: src/main/java/seedu/address/ui/MemberCard.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    @javafx.fxml.FXML
+    private HBox cardPane;
+    @FXML
+    private Label memberName;
+    @FXML
+    private Label id;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
```
Comment (marcuspang) [inline src/main/java/seedu/address/ui/MemberCard.java:35]:
```suggestion
     * Creates a {@code MemberCard} with the given {@code Person} and index to display.
```

File: src/main/java/seedu/address/ui/MemberListPanel.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+public class MemberListPanel extends UiPart<Region> {
+
+    private static final String FXML = "MemberListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(MemberListPanel.class);
+
+    @FXML
+    private ListView<Person> memberListView;
+
+    /**
+     * Creates a {@code TaskListPanel} with the given {@code ObservableList}.
```
Comment (marcuspang) [inline src/main/java/seedu/address/ui/MemberListPanel.java:25]:
```suggestion
     * Creates a {@code MemberListPanel} with the given {@code ObservableList}.
```

File: src/main/java/seedu/address/ui/MemberListPanel.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    public MemberListPanel(ObservableList<Person> memberList) {
+        super(FXML);
+        memberListView.setItems(memberList);
+        memberListView.setCellFactory(listView -> new MemberListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Member} using a {@code MemberCard}.
+     */
+    class MemberListViewCell extends ListCell<Person> {
```
Comment (marcuspang) [inline src/main/java/seedu/address/ui/MemberListPanel.java:36]: Can this be `private static` if it's not being used elsewhere?

File: src/main/java/seedu/address/ui/TaskListPanel.java [hunk 5]
```diff
@@ -41,6 +42,7 @@ protected void updateItem(Task task, boolean empty) {
                 setGraphic(null);
                 setText(null);
             } else {
+                assert task != null;
```
Comment (marcuspang) [inline src/main/java/seedu/address/ui/TaskListPanel.java:45]: Seems a bit redundant

File: src/main/resources/view/LightTheme.css [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+    -fx-text-fill: black;
+    -fx-font-weight: bold;
+    -fx-opacity: 1;
+    -fx-padding: 0 0 10 0;
+}
+
+.label-header {
+    -fx-font-size: 32pt;
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-text-fill: black;
```
Comment (marcuspang) [inline src/main/resources/view/LightTheme.css:32]: The `-fx-font-family` and `-fx-font-weight` doesn't actually affect the UI of the cards because we are already modifying `.cell_big_label` and `.cell_small_label`.

File: src/main/resources/view/LightTheme.css [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+}
+
+.cell_big_label {
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 16px;
+    -fx-text-fill: #010504;
+}
+
+.cell_small_label {
+    -fx-font-family: "Segoe UI Semibold";
```
Comment (marcuspang) [inline src/main/resources/view/LightTheme.css:139]: Can you make the `.cell_small_label` text to be not bold? I.e. remove this `-fx-font-family`

**Review by marcuspang — APPROVED** [review]
Some minor nits, but good to go

File: src/main/java/seedu/address/ui/TaskListPanel.java (hunk 5 above)
Comment (Devanshshah1309) [reply src/main/java/seedu/address/ui/TaskListPanel.java:45]: right haha i was using this for debugging my bad (it actually somehow failed the assertion)

File: src/main/java/seedu/address/ui/MemberListPanel.java (hunk 4 above)
Comment (Devanshshah1309) [reply src/main/java/seedu/address/ui/MemberListPanel.java:36]: I kept it as such to be consistent with `PersonListPanel`

Comment (marcuspang) [reply src/main/java/seedu/address/ui/MemberListPanel.java:36]: Hmm can you make the nested class there `private static` also?

**Comment by marcuspang** [comment]
Also you might want to fix your branch name haha

File: src/main/java/seedu/address/ui/MemberListPanel.java (hunk 4 above)
Comment (Devanshshah1309) [reply src/main/java/seedu/address/ui/MemberListPanel.java:36]: Alright sure

**Comment by Devanshshah1309** [comment]
Ehh branch name doesn't matter 👀 . If everything looks fine now, @marcuspang feel free to merge it.
